src/node/transaction.h

```diff
@@ -16,9 +16,11 @@
  * @param[in]  tx the transaction to broadcast
  * @param[out] &txid the txid of the transaction, if successfully broadcast
  * @param[out] &err_string reference to std::string to fill with error string if available
- * @param[in]  highfee Reject txs with fees higher than this (if 0, accept any fee)
+ * @param[in]  max_tx_fee reject txs with fees higher than this (if 0, accept any fee)
+ * @param[in]  relay flag if both mempool insertion and p2p relay  are requested
+ * @param[in]  wait_callback, wait until callbacks have been processed to avoid stale result due to a sequentially RPC
```

Comment:
Thanks for adding comments. I'd add to the description of the `wait_callback` param that this **must not** be set while cs_main, cs_mempool or cs_wallet is being held or there'll be a deadlock.